README.md

```diff
@@ -848,61 +848,38 @@ if (isDOMNodePresent(node)) {
 
 **[⬆ back to top](#table-of-contents)**
 
-### Avoid conditionals
-
-This seems like an impossible task. Upon first hearing this, most people say,
-"how am I supposed to do anything without an `if` statement?" The answer is that
-you can use polymorphism to achieve the same task in many cases. The second
-question is usually, "well that's great but why would I want to do that?" The
-answer is a previous clean code concept we learned: a function should only do
-one thing. When you have classes and functions that have `if` statements, you
-are telling your user that your function does more than one thing. Remember,
-just do one thing.
+### Replace Nested Conditionals With Guard Clauses
 
 **Bad:**
 
 ```javascript
-class Airplane {
-  // ...
-  getCruisingAltitude() {
-    switch (this.type) {
-      case "777":
-        return this.getMaxAltitude() - this.getPassengerCount();
-      case "Air Force One":
-        return this.getMaxAltitude();
-      case "Cessna":
-        return this.getMaxAltitude() - this.getFuelExpenditure();
+function getPayAmount() {
+  let result;
+  if (isDead) {
+    result = deadAmount();
+  } else {
+    if (isSeparated) {
+      result = separatedAmount();
+    } else {
+      if (isRetired) {
+        result = retiredAmount();
+      } else {
+        result = normalPayAmount();
+      }
     }
   }
+  return result;
 }
 ```
 
 **Good:**
 
 ```javascript
-class Airplane {
-  // ...
-}
-
-class Boeing777 extends Airplane {
-  // ...
-  getCruisingAltitude() {
-    return this.getMaxAltitude() - this.getPassengerCount();
-  }
-}
-
-class AirForceOne extends Airplane {
-  // ...
-  getCruisingAltitude() {
-    return this.getMaxAltitude();
-  }
-}
-
-class Cessna extends Airplane {
-  // ...
-  getCruisingAltitude() {
-    return this.getMaxAltitude() - this.getFuelExpenditure();
-  }
+function getPayAmount() {
```

Comment:
How about this? 

```javascript
  const getPayAmount = () =>
    isDead
      ? deadAmount()
      : isSeparated
      ? separatedAmount()
      : isRetired
      ? retiredAmount()
      : normalPayAmount();
```

Reply:
For the clarity and the purpose of this section, I personally prefer `if statements` with `function`, It can be an alternative solution for that below that, or in a new section, like `try to use ternary operators` when it is possible.